Spec/RealtimeClientConnection.swift

```diff
@@ -250,7 +252,7 @@ class RealtimeClientConnection: QuickSpec {
                             done()
                         case .connected:
                             if let transport = client.internal.transport as? TestProxyTransport, let query = transport.lastUrl?.query {
-                                expect(query).to(haveParam("lib", withValue: "cocoa\(ARTDefault_variant)-1.2.4"))
+                                expect(query).to(haveParam("lib", hasPrefix: "\(ARTDefault_libName)/1.2."))
```

Comment:
This new method is a duplicate of the existing `haveParam` except is uses `hasPrefix` instead of `==`. Im not familiar with Objective-C though. 

I think it does make the test less annoying (you don't have to update the patch version) though by checking for a prefix we are no longer testing this versioning behaviour. 

I think this is not an important detail, feel free to ignore.

Reply:
@ben-xD I'm going to resolve this conversation but feel free to further comment and/or unresolve if you feel there is work to be done. Thanks.

Reply:
@ben-xD @QuintinWillison Sorry for late comment. I've chosen to match "\(ARTDefault_libName)/1.2." because REST request matches this (spec 'RSC7d'), so why match different substrings for the same check?